src/app/containers/header/header.component.ts

```diff
@@ -1,7 +1,8 @@
 import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
```

Comment:
Doesn't look like this is actually used anywhere in the code...?

Reply:
Agreed, removed it now.